pkgs/test/test/runner/runner_test.dart

```diff
@@ -95,6 +95,10 @@ Usage: pub run test [files or directories...]
                                   Currently only supported for browser tests.
 
     --debug                       Runs the VM and Chrome tests in debug mode.
+    --coverage=<directory>        Enables coverage gathering for the Dart VM and outputs an
+                                  lcov formatted file to the specified directory.
```

Comment:
What are the restrictions here? Can the coverage directory be an absolute path?

Reply:
Yes, the coverage directory can be an absolute or relative path. I think I'm going to make this help description as simple as possible and explain how this works more in-depth in the README.

Reply:
@grouma I updated this argument to be super simple, but added a new section to the `README` to describe a bit more how coverage works and what the limitations are, can you give this another look and let me know what you think?